pkg/provisioner/resourcehandlers.go

```diff
@@ -131,6 +139,47 @@ func createObjectBucket(ob *v1alpha1.ObjectBucket, c versioned.Interface, retryI
 	return
 }
 
+func updateObjectBucket(ob *v1alpha1.ObjectBucket, c versioned.Interface, retryInterval, retryTimeout time.Duration) (result *v1alpha1.ObjectBucket, err error) {
+	logD.Info("updating ObjectBucket", "name", ob.Name)
+
+	err = wait.PollImmediate(retryInterval, retryTimeout, func() (bool, error) {
+		result, err = c.ObjectbucketV1alpha1().ObjectBuckets().Update(context.TODO(), ob, metav1.UpdateOptions{})
+		if err != nil {
+			// could be intermittent api error
+			log.Error(err, "probably not fatal, retrying")
+			return false, nil
+		}
+		return true, nil // update successful
+	})
+	return result, err
```

Comment:
in Go since you've named the function returns, you can simply end the func with `return` (or maybe return isn't even required?)

Reply:
I am aware and also prefer the explicit return for readability.